languages/indonesian/pack/src/main/res/xml/indonesian_dictionaries.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Dictionaries>
+    <!-- Dictionary attributes:
+         "nameResId" : the name of the dictionary. Should be short. Reference to a string in values.xml
+         "locale" : the ID of the dictionary's (ISO) locale, e.g., "en", "iw", "ar", "ru", etc. The keyboard should declare the same locale.
+         "id" : UUID for this dictionary. Obtain from here: https://www.guidgenerator.com/
+         "dictionaryResourceId" : a reference to the raw resource of the words dictionary
+         "autoTextResourceId" : a reference to a common typing mistakes XML.
+    -->
+    <Dictionary
+        autoTextResourceId="@xml/indonesian_autotext"
+        description="@string/indonesian_dictionary_description"
+        dictionaryResourceId="@array/indonesian_words_dict_array"
+        id="796adab8-77a1-475f-837a-8c88d25e04f0"
+        initialSuggestions="@array/indonesian_initial_suggestions"

```

Comment:
`indonesian_initial_suggestions` does not exist

Reply:
@menny may I know what the is Initial suggestion ? https://github.com/AnySoftKeyboard/LanguagePack/blob/201be1de7f050524d154fd196640d4e680f63069/languages/english/pack/src/main/res/values/english_pack_strings_dont_translate.xml#L19

Reply:
This is a list of characters that a user might want when there are no suggestions.
For example, ASK will suggest `english_initial_suggestions` before the user types a word (only if the setting is enabled).

Again, I would suggest to remove it.

Reply:
Oh, thank you. I get it now. I really want to know each field. So that I opened https://github.com/AnySoftKeyboard/LanguagePack/issues/191.

Thanks for explanation. Yes I wouldn't like to add it.